tests/scripts/pulp_deb/test_deb_sync_publish.sh

```diff
@@ -54,6 +54,22 @@ expect_succ pulp deb distribution create --name "${ENTITIES_NAME}_distro" \
 expect_succ curl "$curl_opt" --head \
   --fail "$PULP_BASE_URL/pulp/content/cli_test_deb_distro/dists/default/Release"
 
+# Add test for the gpgkey parameter
+expect_succ pulp deb remote update \
+  --name "${ENTITIES_NAME}_remote" \
+  --gpgkey "wrong_key" 
```

Comment:
I see the idea, but this isn't a wrong key, but doesn't remotely look like a key at all.
This would be the exact example of a test suddenly starting to fail i we added server side validation.
I would just not add it instead of creating a second key.

Reply:
True, I will wait for the decision to see if both tests are needed or simply need to be deleted.

may be there is a possibility to rewrite that part to use another valid gpgkey but that is not valid for the test remote